docs/internals.md

```diff
@@ -627,7 +627,7 @@ See [core/Symbols.h] and [core/SymbolRef.h] for more information.
 <!-- Phase descriptions -->
 [parser]: #parser
 [desugar]: #desugar
-[dsl]: #dsl
+[dsl]: #syntax_transform
```

Comment:
Tiny nit: this `dsl` should probably change, too.

Reply:
great catch!